src/components/application_manager/include/application_manager/commands/command_notification_impl.h

```diff
@@ -51,7 +51,7 @@ class CommandNotificationImpl : public CommandImpl {
   virtual bool Init();
   virtual bool CleanUp();
   virtual void Run();
-  void SendNotification();
+  void SendNotification(const bool final_message = false);
```

Comment:
New parameter require description

Reply:
@LuxoftAKutsan Done in c0352a9a390d3c4cfe97d88a53af79ddb2eb9a33